snap/snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: sabnzbd
+version: git
+summary: SABnzbd
+description: The automated Usenet download tool
+grade: stable
+confinement: strict
+
+apps:
+  sabnzbd:
+    command: opt/sabnzbd/SABnzbd.py -f $SNAP_COMMON
+    daemon: simple
+    plugs: [network, network-bind]
+
+parts:
+  sabnzbd:
+    plugin: python
+    source: .
+    python-version: python2
+    python-packages: [cheetah3, cryptography, sabyenc]
+    stage-packages: [par2, unrar, p7zip-full, unzip]
```

Comment:
Recently we removed the need for `unzip`, when there's no `unzip` we just use `p7zip`.

How does it get these packages? Is there a way to add `par2-mt` from our own PPA of `par2`?

Reply:
Yeah I am happy to change the the dependencies around, I am unfamiliar with what the preferred versions and installation method is (apt vs pip).

Reply:
It gets the packages from the Ubuntu repositories, I am not sure there is a good way to add a PPA unfortunately.